src/charm.py

```diff
@@ -301,9 +301,7 @@ def _gen_wp_config(self):
 
         replica_relation_data = self._replica_relation_data()
         for secret_key in self._wordpress_secret_key_fields():
-            secret_value = replica_relation_data.get(secret_key)
-            if not secret_value:
-                raise ValueError(f"{secret_key} value is empty")
+            secret_value = replica_relation_data[secret_key]
```

Comment:
Is there a guarantee that `secret_value` is empty/ None? Just curios since the check for it was removed

Reply:
Instead of using `get`, I am using `__getitem__` which will throw an error when the item does not exist. So the none check here is not needed anymore.

Reply:
Makes sense, would it be worth handling that error and giving feedback to the user? I'm guessing the charm would go into error state if `secret_key` is not in `replica_relation_data`?

Reply:
It shouldn't happen, there must be some unknown internal error if the replica_relation_data doesn't contain the secret keys. So I think it's the best to let it throw as an error.